addons/it/adecf/adecf.go

```diff
@@ -0,0 +1,55 @@
+// Package adecf handles the validation rules in order to use
+// GOBL with the Italian Agenzia delle Entrate format.
+package adecf
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+)
+
+// Key to identify the AdE adecf addon
+const (
+	// V1 for AdE format
+	V1 cbc.Key = "it-adecf-v1"
```

Comment:
The `adecf` name looks highly unreadable in this context. Would `it-ticket-v1` work?